samples/AvatarWithRoundedCorner/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿
+
+namespace AvatarWithRoundedCorner
+{
+    using System;
+    using System.Numerics;
+    using ImageSharp;
+    using SixLabors.Shapes;
+
+    class Program
+    {
+        static void Main(string[] args)
```

Comment:
The great thing about source control is we can always change it in the future. It's not a blocker for this PR as far as I'm concerned.